crates/egui/src/data/key.rs

```diff
@@ -25,6 +25,17 @@ pub enum Key {
     PageUp,
     PageDown,
 
+    CapsLock,
+    NumLock,
+    ScrollLock,
+
+    Alt,
+    AltRight,
+    Control,
+    ControlRight,
+    Shift,
+    ShiftRight,
```

Comment:
A) it's weird that there is both `Alt` and `AltRight`, but not `AltLeft`
B) on the PR preview (https://egui-pr-preview.github.io/pr/5400-patch150/) the `Right` versions doesn't work at all.

I think we either should just have a single `Alt`, OR we should have `AltLeft` and `AltRight`, and make sure it works everywhere